requests_oauthlib/oauth2_session.py

```diff
@@ -337,6 +338,10 @@ def fetch_token(
             request_kwargs["params"] = dict(urldecode(body))
         else:
             raise ValueError("The method kwarg must be POST or GET.")
+            
+        for hook in self.compliance_hook["access_token_request"]:
```

Comment:
Is this access_token_request declared somewhere?

Reply:
Good point 🤦 I've corrected this.

I'm not sure the best way to go about a unit test for these as they don't change the response like the other compliance hooks. Any suggestions or advice would be appreciated. Thank you.